pinot-clients/pinot-java-client/src/main/java/org/apache/pinot/client/utils/BrokerSelectorUtils.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.client.utils;
+
+import java.util.List;
+import java.util.Objects;
+
+
+public class BrokerSelectorUtils {
+
+  private BrokerSelectorUtils() {
+  }
+
+  /**
+   *
+   * @param tablesBrokersList: List of brokers hosting each table in the list.
+   * @return list of common brokers hosting all the tables.
+   */
+  public static List<String> getTablesCommonBrokers(List<List<String>> tablesBrokersList) {
```

Comment:
do we plan to move more utils into this class?

Reply:
Yes, that's my plan. Eventually, we will move all common functionality to this class.

Reply:
yes let's add a 
```
public static List<String> getTablesCommonBrokers(List<String> tableNames) {
  // ... 
}
```